internal/summoner/acquire/headlessNG.go

```diff
@@ -316,7 +230,7 @@ func PageRender(v1 *viper.Viper, mc *minio.Client, logger *log.Logger, timeout t
 			});
 		}
 
-		function retry(fn, retriesLeft = 3, interval = 3000) {
+		function retry(fn, retriesLeft = 3, interval = 1000) {
```

Comment:
Did you mean to make that 1000? I think @valentinedwv had it at 3000, or am I misremembering?